website/docs/about/introduction/orca-intro.md

```diff
@@ -14,6 +14,8 @@ ORCA Working Group.
 
 ## How do I implement or use ORCA?
 
-Some words here to talk about Cumulus integration/deployment at a very high level.
+Information on implementing ORCA is located here or in the
+[ORCA GitHub repository](https://github.com/nasa/cumulus-orca).
 
+Information on ORCA use for Operators and Data Managers is located here.
```

Comment:
Need to create a link on **here** to the stub out of operator documents.

Reply:
Done.